pages/api/program.ts

```diff
@@ -19,27 +16,17 @@ export default async function handler(
     default:
       return res.status(405).end();
   }
-
 }
 
-const getPrograms = async (req: NextApiRequest, res: NextApiResponse) => {
+const getProgram = async (req: NextApiRequest, res: NextApiResponse) => {
+  const { id } = req.query; 
   try {
-    const programs = await prisma.program.findMany();
-    return res.status(200).json(programs);
-  } catch (error) {
-    return res.status(500).json({error});
-  }
-}
-
-const getProgramById = async (req: NextApiRequest, res: NextApiResponse) => {
-  const { id } = req.query;
-  try {
-    const program = await prisma.program.findUnique({
-      where: { id: Number(id) }
+    const program  = await prisma.program.findFirst({
+      where: { id: Number(id) },
     });
     return res.status(200).json(program);
   } catch (error) {
-    return res.status(500).json({error}); 
+    return res.status(500).json({error});
```

Comment:
Nit: should add spaces inside braces 

`return res.status(500).json({ error });`

FYI 'NIT' is basically a nitpicky comment, very minor style suggestion, can ignore.

Reply:
I think learning good code styling is absolutely worthwhile, so thank you for pointing it out!